packages/client/docusaurus/docs/javascript/02-guides/03-call-and-participant-state.mdx

```diff
@@ -18,48 +24,37 @@ This approach makes it possible to access the state and be notified about change
 
-Here are all the call state properties:
+Here are is an excerpt of the call state properties:
 
 | Reactive value           | Static value            | Description                                                                                                  |
 | ------------------------ | ----------------------- | ------------------------------------------------------------------------------------------------------------ |
 | `members$`               | `members`               | The list of call members                                                                                     |
 | `callingState$`          | `callingState`          | Provides information about the call state. For example, `RINGING`, `JOINED` or `RECONNECTING`.               |
+| `localParticipant$`      | `localParticipant`      | The local participant is the logged-in user.                                                                 |
+| `remoteParticipants$`    | `remoteParticipants`    | All participants except the local participant.                                                               |
+| `participants$`          | `participants`          | All participants, including local and remote participants.                                                   |
 | `startedAt$`             | `startedAt`             | The actual start time of the current call session.                                                           |
 | `callStatsReport$`       | `callStatsReport`       | When stats gathering is enabled, this observable will emit a new value at a regular (configurable) interval. |
-| `metadata$`              | `metadata`              | The `CallResponse`, see below for more information.                                                          |
-| `callRecordingList$`     | `callRecordingList`     | The latest list of recordings performed during the call.                                                     |
 | `hasOngoingScreenShare$` | `hasOngoingScreenShare` | It will return `true` if at least one participant is sharing their screen.                                   |
+| `custom$`                | `custom`                | Custom data attached to the call.                                                                            |
 | `dominantSpeaker$`       | `dominantSpeaker`       | The participant that is the current dominant speaker of the call.                                            |
 | `ownCapabilities$`       | `ownCapabilities`       | The capabilities of the local participant.                                                                   |
 
-The `CallResponse` object contains the following information:
-
-| Name               | Description                                             |
-| ------------------ | ------------------------------------------------------- |
-| `backstage`        | It's `true` if the call is in backstage mode.           |
```

Comment:
Shouldn't we add the properties of `CallResponse` to the above table?

Reply:
I'll add them now for both React and Plain-JS.

Reply:
Fixed with [efbb2de](https://github.com/GetStream/stream-video-js/pull/931/commits/efbb2dedc45e0b5730bc2dd5517ddc89efd2b365)